_posts/2024-09-27-learning-a-programming-language-with-generative-ai.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: Learning a programming language with generative ai
+author: andandersson
+excerpt: >
+  Experiences from learning a programing language with generatve ai.
+tags:
+ - Generative ai
+ - C#
+ - .net
```

Comment:
canonicalize to "DOTNET" that is used in other posts

Reply:
corrected